src/UI/Interviewer/WB.UI.Interviewer/Activities/Dashboard/DashboardActivity.cs

```diff
@@ -134,17 +135,57 @@ private void CreateFragments()
                 new PropertyChangedEventArgs(itemsCountPropertyCountName));
 
             var tabLayout = this.FindViewById<TabLayout>(Resource.Id.tabs);
-            tabLayout.SetupWithViewPager(this.viewPager);
+
+            new TabLayoutMediator(tabLayout, this.viewPager, 

```

Comment:
save TabLayoutMediator and TabConfigurationStrategy  to objects and dispose its then